mne/time_frequency/multitaper.py

```diff
@@ -458,9 +458,12 @@ def tfr_array_multitaper(epoch_data, sfreq, freqs, n_cycles=7.0,
                          zero_mean=True, time_bandwidth=None, use_fft=True,
                          decim=1, output='complex', n_jobs=1,
                          verbose=None):
-    """Compute time-frequency transforms using wavelets and multitaper windows.
+    """Compute Time-Frequency Representation (TFR) using DPSS tapers.
 
-    Uses Morlet wavelets windowed with multiple DPSS tapers.
+    Same computation as the function tfr_multitaper.
+    Difference: tfr_multitaper takes epochs in the mne.Epochs format as input.
+    This function, tfr_array_multitaper, takes epochs in the numpy.array format
+    as input.
```

Comment:
```suggestion
    Same computation as `~mne.time_frequency.tfr_multitaper`, but operates on
    :class:`NumPy arrays <numpy.ndarray>` instead of `~mne.Epochs` objects.
```